core/lib/generators/refinery/cms/cms_generator.rb

```diff
@@ -75,20 +75,33 @@ def append_gitignore!
     end
 
     def append_heroku_gems!
-      append_file 'Gemfile', %q{
+      append_file 'Gemfile', %Q{
+# The Ruby version is specified here so that Heroku uses the right version.
+ruby #{ENV['RUBY_VERSION'].inspect}
+
 # The Heroku gem allows you to interface with Heroku's API
 gem 'heroku'
 
-# Fog allows you to use S3 assets (added for Heroku)
-gem 'fog'
+group :production do
+  # Fog allows you to use S3 assets (added for Heroku)
```

Comment:
Oh, definitely outdated, it should replace the word `Fog` with `Dragonfly's S3 Data Store extension`

Are you able to send a PR against this branch?